pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -277,6 +277,19 @@ func (ds *dockerService) StopPodSandbox(ctx context.Context, r *runtimeapi.StopP
 	}
 
 	// TODO: Stop all running containers in the sandbox.
```

Comment:
Nit: If we end up merging this change, could you please delete the TODO.

Reply:
Updated
Thanks